src/train.py

```
@@ -0,0 +1,128 @@
+    early_stopper = EarlyStopping(monitor='val_accuracy', patience=10,
+                                  restore_best_weights=True, verbose=1)
+
+    model_checkpoint_path = os.path.join(RESULTS_DIR, f"{MODEL_NAME}_best.keras")
+    model_checkpoint = ModelCheckpoint(model_checkpoint_path, monitor='val_accuracy',
+                                       save_best_only=True, verbose=1)
+
+    # --- 5. Tanítás ---
+    print(f"\n--- Tanítás indítása ({args.epochs} epoch) ---")
+
+    if args.no_augmentation or args.model == 'mlp':
+        if args.model != 'mlp': print("Adatbővítés KIkapcsolva.")
```

Ruff (0.14.5)

Multiple statements on one line (colon)

(E701)